stdlib/system/build.gradle

```diff
@@ -39,6 +39,10 @@ dependencies {
     baloImplementation project(path: ':ballerina-time', configuration: 'baloImplementation')
     baloImplementation project(path: ':ballerina-lang:internal', configuration: 'baloImplementation')
     baloImplementation project(path: ':ballerina-io', configuration: 'baloImplementation')
+    baloImplementation project(path: ':ballerina-java', configuration: 'baloImplementation')
+
+    interopImports project(':ballerina-system')
```

Comment:
This is the system module gradle so why do you need this `interopImports project(':ballerina-system')` here?